components/Divider/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,73 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS Components/Data Display/Divider"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    docs: { source: { type: "dynamic" } },
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Divider" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <p>I am above the divider.</p>
+    <hr class="denhaag-divider" role="presentation" />
+    <p>I am below the divider.</p>
+  </Story>
+  {/* eslint-enable react/no-unknown-property */}
+</Canvas>
+
+### Vertical
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Vertical">
+    <div style={{ display: "flex" }}>
```

Comment:
@paulAcato I think it's cleaner to assign a css class to the `<div>` instead of inline css.

Reply:
Although I would usually agree, since this is just a single style for storybook demo purposes only, I don't mind an inline style to set it to flex.